pkg/osbuild/bootupd_gen_metadata_stage.go

```diff
@@ -0,0 +1,7 @@
+package osbuild
+
+func NewBootupdGenMetadataStage() *Stage {
+	return &Stage{
```

Comment:
Again, incredible silly, but I would love to see a test here if nothing else then to ensure that if we change the way we do stuff (e.g. auto-generate stages from python directly) then the tests will confirm that things are still generated the right way.